numpy/polynomial/polynomial.py

```diff
@@ -1441,7 +1441,7 @@ class Polynomial(ABCPolyBase):
 
     The Polynomial class provides the standard Python numerical methods
     '+', '-', '*', '//', '%', 'divmod', '**', and '()' as well as the
-    attributes and methods listed in the `ABCPolyBase` documentation.
+    attributes and methods listed in the ``ABCPolyBase`` documentation.
```

Comment:
The link was correct as is - the problem is the thing its trying to link to doesn't exist!